### PR TITLE
Corrected grammar in App Center description

### PR DESCRIPTION
**Pull request by imasaru**
@artemanufrij @ryonakano Let's try this again!

I tried to fix all of the grammar-related issues that I could find. I hope this helps! I also explained my corrections in line comments.

### CODE REVIEW

File: data/com.github.artemanufrij.playmymusic.appdata.xml.in [hunk 1]
```diff
@@ -5,18 +5,18 @@
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-3.0+</project_license>
   <name>Melody</name>
-  <summary>A music player for listening to local music files, online radios and audio CD's</summary>
```
Comment (imasaru) [inline data/com.github.artemanufrij.playmymusic.appdata.xml.in]: The noun "radio" can refer to both a radio station and a device that can receive radio signals (a radio). When pluralized, it most likely refers to the device or devices and not a single station.

**It seems like you want to refer to a collective body of all possible online radio stations. To do so, you would say online radio.**

File: data/com.github.artemanufrij.playmymusic.appdata.xml.in [hunk 2]
```diff
@@ -5,18 +5,18 @@
...
+  <summary>A music player for listening to local music files, online radio, and audio CDs</summary>
   <description>
-    <p>A very fast music player designed extra for large local libraries which also supports online radios and downloading album covers</p>
+    <p>A very fast music player designed for extra large local libraries that supports online radio and downloading album covers.</p>
     <p>Features:</p>
     <ul>
-      <li>Show all Albums from your library</li>
```
Comment (imasaru) [inline data/com.github.artemanufrij.playmymusic.appdata.xml.in]: I can tell that you speak German—the German language tends to capitalize some nouns, correct me if I am wrong. However, this practice is not done in English 😊

File: data/com.github.artemanufrij.playmymusic.appdata.xml.in [hunk 3]
```diff
@@ -5,18 +5,18 @@
...
-      <li>Group tracks by Artists</li>
-      <li>Manage your Playlists</li>
-      <li>Listen to online Radio</li>
+      <li>Show all albums from your library</li>
+      <li>Group tracks by artists</li>
+      <li>Manage your playlists</li>
+      <li>Listen to online radio</li>
       <li>Audio CD support</li>
       <li>Manage your MTP device</li>
-      <li>ID3-Tag support</li>
```
Comment (imasaru) [inline data/com.github.artemanufrij.playmymusic.appdata.xml.in]: This is not a point of English but of how the ID3 organization stylizes "ID3" on their website (id3.org)

**Comment by imasaru** [comment]
@artemanufrij Forgive me, I should have commented on the corrections in the *green* line instead of the red. I'm not removing lines wholesale, sorry.

**Comment by artemanufrij** [comment]
@imasaru  thank you for your support!

**Pull request merged**